src/github.com/couchbase/sync_gateway/rest/config.go

```diff
@@ -278,6 +285,9 @@ func (self *ServerConfig) MergeWith(other *ServerConfig) error {
 	if self.Facebook == nil {
 		self.Facebook = other.Facebook
 	}
+  if self.CORS == nil {
+  	self.CORS = other.CORS
+  }
```

Comment:
Can you run go fmt for this PR?  Looks like there are a few places that need cleaned up.